app/src/main/java/com/napzak/market/core/designsystem/component/button/EnableDisableTextButton.kt

```diff
@@ -0,0 +1,65 @@
+package com.napzak.market.core.designsystem.component.button
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+
+/**
+ * 활성화/비활성화 버튼 컴포넌트
+ *
+ * @param text 버튼에 표시될 텍스트
+ * @param isEnabled 버튼 활성화 여부
+ * @param onClick 버튼 클릭 시 수행할 동작
+ */
+
+@Composable
+fun EnableDisableTextButton(
+    text: String,
+    isEnabled: Boolean,
+    onClick: () -> Unit
+) {
+    CommonButton(
+        text = text,
+        onClick = { if (isEnabled) onClick() },
+        isEnabled = isEnabled,
+        width = 335.dp,
+        height = 52.dp,
```

Comment:
- `EnableDisableTextButton`도 생성자에 modifier를 받아서 그대로 `CommonButton`의 생성자에 들어가도록 구현하면 `EnableDisableTextButton`을 사용하는 화면에서 직접 크기를 지정해줄 수 있습니다!

```suggestion
fun EnableDisableTextButton(
    text: String,
    isEnabled: Boolean,
    onClick: () -> Unit,
    modifier: Modifier = Modifier,
) {
    CommonButton(
        text = text,
        onClick = { if (isEnabled) onClick() },
        isEnabled = isEnabled,
        modifier = modifier,
```

Reply:
헉 손수 코드 수정을 적어주시다니..감사합니다